tests/integration/conftest.py

```diff
@@ -41,12 +42,14 @@ def observer_id(request):
 
 @pytest.fixture(scope="session")
 def connection(host):
-    return _sdk.from_local_account(host, os.environ["C42_USER"], os.environ["C42_PW"])
+    user = os.environ.get("C42_USER") or "test.user@example.com"
+    pw = os.environ.get("C42_PW") or "password"
+    return _sdk.from_local_account(host, user, pw)
```

Comment:
Can we fail fast here so we don't have to wait out all the failures if the mockserver isn't up or we provide bad creds? Something like:

```
    try:
        return _sdk.from_local_account(host, user, pw)
    except Exception as e:
        pytest.exit("Failed to init sdk for integration tests: {e}".format(e))
```

Reply:
Done